database/neo4j/neo4j.go

```diff
@@ -256,7 +255,16 @@ func (n *Neo4j) ensureVersionConstraint() (err error) {
 		}
 	}()
 
-	query := fmt.Sprintf("CREATE CONSTRAINT ON (a:%s) ASSERT a.version IS UNIQUE", n.config.MigrationsLabel)
+	// Get constraint and check to avoid error duplicate
+	res, err := neo4j.Collect(session.Run(fmt.Sprintf("CALL db.constraints() yield name WHERE name=\"%s\" RETURN *", DefaultVersionUniqueKey), nil))
```

Comment:
Looks like `YIELD` is a [keyword](https://neo4j.com/docs/cypher-manual/current/clauses/call/#call-call-a-procedure-within-a-complex-query-using-call-yield) so make it all-caps.

Also, let's `RETURN name` to reduce the amount of data that's returned.

Reply:
Hi @dhui 

This issue because of the version, Neo4J 3 not support it
I use labels instead to support Neo4J 3 and 4